odoo/addons/base/models/res_users.py

```diff
@@ -537,6 +537,9 @@ def _search(self, args, offset=0, limit=None, order=None, count=False, access_ri
 
     @api.model_create_multi
     def create(self, vals_list):
+        for values in vals_list:
+            if (not values.get('signature') or tools.is_html_empty(values['signature'])) and values.get('name'):
```

Comment:
Should we rather test `if not 'signature' in values` to allow people passing specifically False if they don't want one?